pkg/products/amqonline/reconciler.go

```diff
@@ -207,8 +216,8 @@ func (r *Reconciler) createResource(ctx context.Context, resourceName string, se
 	return resource, nil
 }
 
-func (r *Reconciler) reconcileTemplates(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
-	// Interate over template_list
+func (r *Reconciler) reconcileTemplates(ctx context.Context, installation *integreatlyv1alpha1.Installation, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
I'd put this back for now, I think it might be what's causing the failures.

Reply:
hmm, that seemed like a typo, hence the change :thinking:

Reply:
Fixed, it was the `installation` in the functions signature